.github/workflows/cloudflare.yml

```diff
@@ -40,9 +40,9 @@ jobs:
           echo "CURRENT_BRANCH_NAME=${{ github.ref_name }}" >> ${GITHUB_ENV}
       - name: Determine Build Mode
         run: |
-          if [[ "${{ github.ref_name }}" == "main" || "${{ github.ref_name }}" == "release" ]]; then
+          if [[ "${{ github.ref_name }}" == "main" || "${{ github.ref_name }}" == "release" || "${{ github.head_ref }}" == "main" || "${{ github.head_ref }}" == "release" ]]; then
             echo "BUILD_MODE=production" >> ${GITHUB_ENV}
-          elif [[ "${{ github.ref_name }}" == "private" ]]; then
+          elif [[ "${{ github.ref_name }}" == "private" || "${{ github.head_ref }}" == "private" ]]; then
```

Comment:
q: do we want to also use `base_ref` instead of `ref_name` on the left or would that be a rug?

Reply:
On the left? It's one or the other:

- If the CI has been triggered by a commit, it use `ref_name`
- If it has been triggered by a pull request opening, it use `head_ref`

Conslusion: we need both